apps/sms/js/information.js

```diff
@@ -222,7 +220,7 @@ var VIEWS = {
       });
     },
 
-    elements: ['contact-list']
+    elements: ['contact-list', 'header', 'header-text', 'container']
```

Comment:
Yup, we don't need container in participants view.I will remove it in HTML also as you said.